app/src/main/java/org/oppia/android/app/application/alphakenya/AlphaKenyaApplicationComponent.kt

```diff
@@ -89,7 +89,7 @@ import javax.inject.Singleton
     LoggingIdentifierModule::class, ApplicationLifecycleModule::class,
     NetworkConnectionDebugUtilModule::class, LoggingIdentifierModule::class,
     SyncStatusModule::class, LogReportingModule::class, NetworkConnectionUtilProdModule::class,
-    HintsAndSolutionProdModule::class
+    HintsAndSolutionProdModule::class, PlatformParameterModule::class
```

Comment:
This is going to cause Dagger conflicts, I think, since the Kenya-specific version of the alpha build has its own platform parameter module. You need to update that rather than add the shared one here.

Reply:
could you please tell what is the file name for this?

Reply:
added an updated logic here, but more attention is needed here.

Reply:
done